contracts/modules/v2/StakingPool.sol

```diff
@@ -408,28 +437,40 @@ contract StakingPool is IStakingPool, ERC721 {
   /* pool management */
 
   function setProductDetails(ProductParams[] memory params) external onlyManager {
-
+    // silence compiler warnings
+    params;
+    activeStake = activeStake;
+    revert("Not implemented");
```

Comment:
I think it's less likely to block others when integrating these function if you use a todo comments instead of a revert. It's quite hard to miss if we forget implementing something here.